sklearn/model_selection/tests/test_split.py

```diff
@@ -908,6 +844,14 @@ def test_cv_iterable_wrapper():
     # Check if get_n_splits works correctly
     assert_equal(len(cv), wrapped_old_skf.get_n_splits())
 
+    kf_iter = KFold(n_splits=5).split(X, y)
+    kf_iter_wrapped = check_cv(kf_iter)
+    # Since the wrapped iterable is enlisted and stored,
+    # split can be called any number of times to produce
+    # consistent results
```

Comment:
Only true because it is not random. Please note this. Indeed, it may be good to test that randomization produces different splits in repeated calls to `split`.